src/collection-preferences/interfaces.ts

```diff
@@ -169,10 +169,14 @@ export interface CollectionPreferencesProps<CustomPreferenceType = any> extends
    * Called when the user confirms a preference change using the confirm button in the modal footer.
    *
    * The event `detail` contains the following:
+   * - `contentDensity` (boolean) - (Optional) The current content density preference value. Available only if you specify the `contentDensityPreference` property.
```

Comment:
Reordered the items alphabetically as we start to have a respectable number of them (this PR increases them from 4 to 8)